CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Added limited Fuchsia support (#[1285](https://github.com/nix-rust/nix/pull/1285))
 - Added `getpeereid` (#[1342](https://github.com/nix-rust/nix/pull/1342))
 ### Fixed
+- TimerFd now closes the underlying fd on drop.
```

Comment:
Also, in the "Removed" section you should note that is no longer Clone and Copy.

Reply:
Done